EDEngineer.Tests/ReferenceDataIntegrityTests.cs

```diff
@@ -58,6 +58,7 @@ public void No_engineer_mistyped()
             {
                 "@Synthesis",
                 "@Technology",
+                "@Merchant",
```

Comment:
this is already at the bottom of the list

Reply:
Done